web/src/lib/components/faces-page/people-list.svelte

```diff
@@ -4,22 +4,26 @@
   import SearchPeople from '$lib/components/faces-page/people-search.svelte';
   import { t } from 'svelte-i18n';
 
-  export let screenHeight: number;
-  export let people: PersonResponseDto[];
-  export let peopleToNotShow: PersonResponseDto[];
-  export let onSelect: (person: PersonResponseDto) => void;
+  interface Props {
+    screenHeight: number;
+    people: PersonResponseDto[];
+    peopleToNotShow: PersonResponseDto[];
+    onSelect: (person: PersonResponseDto) => void;
+  }
+
+  let { screenHeight, people, peopleToNotShow, onSelect }: Props = $props();
 
-  let searchedPeopleLocal: PersonResponseDto[] = [];
+  let searchedPeopleLocal: PersonResponseDto[] = $state([]);
 
-  let name = '';
-  let showPeople: PersonResponseDto[];
+  let name = $state('');
+  let showPeople: PersonResponseDto[] = $state([]);
 
-  $: {
+  $effect(() => {
     showPeople = name ? searchedPeopleLocal : people;
     showPeople = showPeople.filter(
       (person) => !peopleToNotShow.some((unselectedPerson) => unselectedPerson.id === person.id),
     );
-  }
+  });
```

Comment:
This effect causes an infinite loop
```suggestion

  const showPeople = $derived(
    (name ? searchedPeopleLocal : people).filter(
      (person) => !peopleToNotShow.some((unselectedPerson) => unselectedPerson.id === person.id),
    ),
  );
```